rdkit/Chem/Fingerprints/ClusterMols.py

```diff
@@ -39,7 +39,7 @@ def GetDistanceMatrix(data, metric, isSimilarity=1):
 
   """
   nPts = len(data)
-  res = numpy.zeros((nPts * (nPts - 1) / 2), numpy.float)
+  res = numpy.zeros(int(nPts * (nPts - 1) / 2), numpy.float)
```

Comment:
I think it's better to just use integer division here: `(nPts * (nPts - 1) // 2)`